hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -778,7 +793,7 @@ async def _update_order_status(self):
                         OrderUpdate(
                             client_order_id=client_order_id,
                             exchange_order_id=exchange_order_id,
-                            trading_pair=await OBDS.trading_pair_associated_to_exchange_symbol(order_data["symbol"]),
+                            trading_pair=await AscendExAPIOrderBookDataSource.trading_pair_associated_to_exchange_symbol(order_data["symbol"]),
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols